src/scripts/main.js

```diff
@@ -1,3 +1,374 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tbody = table.querySelector('tbody');
+
+// #region sorting
+
+const headings = document.querySelectorAll('th');
+const footerHeadings = document.querySelectorAll('tfoot th');
+const rows = [...tbody.querySelectorAll('tr')];
+
+const sortDirections = Array.from({ length: headings.length }, () => 1);
```

Comment:
Just fix it to pass all tests
```suggestion
const sortDirections = Array.from({ length: headings.length }, () => -1);
```

Reply:
let`s go!!! thanks)))